Scalar.FunctionalTests/Tools/ScalarHelpers.cs

```diff
@@ -16,8 +16,7 @@ public static string GetObjectsRootFromGitConfig(string repoRoot)
 
         public static string GetInternalParameter()
         {
-            return $"\"{{\\\"ServiceName\\\":\\\"{ScalarServiceProcess.TestServiceName}\\\"," +
-                    "\\\"StartedByService\\\":false}\"";
+            return "\"{}\"";
         }
```

Comment:
I think we should be able to delete this entirely.

Reply:
Let's try that.